test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -73,8 +66,7 @@ public GivenThatICareAboutMultilevelSDKLookup()
 
             // Set a dummy framework version (9999.0.0) in the cwd sharedFx location. We will
             // always pick the framework from cwd to avoid interference with the sharedFxLookup
```

Comment:
More comments that need to be reworked.  Please scrub all the comments in the test code.